packages/arb-token-bridge-ui/src/components/MainContent/MainContent.tsx

```diff
@@ -32,7 +32,10 @@ export const useMainContentTabs = create<MainContentTabStore>(set => ({
 export function MainContent() {
   const [isArbitrumStatsVisible] =
     useLocalStorage<boolean>(statsLocalStorageKey)
-  const { selectedTab, setSelectedTab } = useMainContentTabs()
+  const { selectedTab, setSelectedTab } = useMainContentTabs(state => ({
+    selectedTab: state.selectedTab,
+    setSelectedTab: state.setSelectedTab
+  }))
```

Comment:
When we have multiple property returned, we have 2 options, using `shallow` like here, or calling `useMainContentTabs` multiple time:
```
const selectedTab = useMainContentTabs(state => state.selectedTab)
const setSelectedTab = useMainContentTabs(state => state.setSelectedTab)
```

Reply:
See https://github.com/OffchainLabs/arbitrum-token-bridge/pull/2365#issuecomment-2765728569

Multiple lines is easier to enforce with eslint rule.